tests/unit/plugins/kubernetes/kubernetes_pod_executor_test.py

```diff
@@ -15,6 +15,7 @@
 from kubernetes.client import V1SecurityContext
 from kubernetes.client import V1Volume
 from kubernetes.client import V1VolumeMount
+from kubernetes.client.models.v1_container_port import V1ContainerPort
```

Comment:
Oh, weird, how come we had to import from `models` unlike all our other V1* objects (V1VolumeMount, V1SecurityContext, etc)

Reply:
oh, my bad - this was my LSP importing the full path instead of the "nice" path...lemme fix this